dss/dynamodb/__init__.py

```diff
@@ -7,16 +7,18 @@ class DynamoDBItemNotFound(Exception):
     pass
 
 
-def _format_item(hash_key, sort_key, value):
+def _format_item(hash_key: str, sort_key: str, value: str, ttl: int=None):
```

Comment:
[This comment](https://github.com/HumanCellAtlas/data-store/pull/2363/files#r315452815) about using `typing.Optional[<type>]` applies here too